lib/misc.py

```diff
@@ -33,6 +33,15 @@ def __call__(self, *args, **kwargs):
 # Argument: command line arguments Namespace. Do not need to call sys.exit()
 # because caller manages that.
 
+def delete(cli):
+   img = ch.Storage(cli.storage)
+   img_ref = ch.Image_Ref()
+   img_ref.name = cli.image_ref
+   if(img.image_unpacked_p(img_ref)):
+      ch.rmtree(img.unpack(img_ref))
+   else:
+      ch.ERROR("image %s doesn't exist" % cli.image_ref)	
```

Comment:
This logic should be a little more complicated I think. I'd recommend the following algorithm:

1. Check if the image's unpack directory exists. If not, that's the error you have already, though for style I'd say "not found" and use `ch.FATAL`.
2. Check if the image's unpack directory is an image. If not, that's an inconsistency in the storage directory and a different error.
3. Delete the unpack directory.
4. Report successful deletion.

You might consider making step 1, and then steps 2 and 3, into two methods inside `Image`. This will make it easier to update them with the build cache later.